include/rtc/plihandler.hpp

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2023 Arda Cinar
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef RTC_PLI_RESPONDER_H
+#define RTC_PLI_RESPONDER_H
+
+#if RTC_ENABLE_MEDIA
+
+#include "mediahandlerelement.hpp"
+#include "utils.hpp"
+#include <functional>
+
+namespace rtc {
+
+/// Responds to PLI and FIR messages sent by the receiver. The sender should respond to these
+/// messages by sending an intra. 
+class RTC_CPP_EXPORT PliHandler final : public MediaHandlerElement {
+    std::function<void(void)> mOnPli;
```

Comment:
This should be changed to `synchronized_callback` as it's the actual callback being called by this class.